PolicyProject1/src/main/java/com/example/demo/model/PaymentTerm.java

```diff
@@ -0,0 +1,5 @@
+package com.example.demo.model;
+
+public enum PaymentTerm {
```

Comment:
here we have created enum PaymentTerm .Enum is a special "class" that represents a group of constants.